### PR TITLE
Feature/update design preferences

### PR DESCRIPTION
**Pull request by alexsicart**

### CODE REVIEW

File: src/components/preferences/UserInfo.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { addUser } from '../../actions';
-
+import Bootstrap from './bootstrap.css';
+import './style.css';
 
 class UserInfo extends React.Component {
-// Trying to get data from fetch ?using props?
+  // Trying to get data from fetch ?using props?
```
Comment (jmporchet) [inline src/components/preferences/UserInfo.js]: Have you solved this question?

**Review by jmporchet — COMMENTED** [review]
Check my comment for file src/components/preferences/UserInfo.js

**Comment by alexsicart** [comment]
This pull request has been reviewed and works correctly, waiting for @jmporchet @arol review.

AS

**Review by arol — APPROVED** [review]

**Comment by alexsicart** [comment]
Completed.

AS

**Pull request merged**